crates/turborepo-lib/src/run/mod.rs

```diff
@@ -215,10 +215,20 @@ impl<'a> Run<'a> {
                 sock_file: self.base.daemon_file_root().join_component("turbod.sock"),
             };
 
-            let client = connector.connect().await?;
-            debug!("running in daemon mode");
-            daemon = Some(client);
-        }
+            match connector.connect().await {
+                Ok(client) => {
+                    debug!("running in daemon mode");
+                    Some(client)
+                }
+                Err(e) => {
+                    warn!("failed to connect to daemon {e}");
```

Comment:
Not blocking, but do we want this to be displayed to users at the default log level?

Reply:
Good point, I'll drop this to debug